tests/recorder/recorder.go

```diff
@@ -0,0 +1,233 @@
+package main
```

Comment:
Oh nice! Thanks for sharing. This looks like a great option for "not fancy" prompts. And I agree goscillo seems a bit... forced. Naming things is hard, `survey` used to be called `probe` which changed for obvious reasons haha